src/colmap/estimators/alignment_test.cc

```diff
@@ -126,8 +126,8 @@ TEST(Alignment, MergeReconstructions) {
   Reconstruction tgt_reconstruction = src_reconstruction;
 
   // Remove the camera of the first image from the target reconstruction
-  const std::vector<image_t>& image_ids = tgt_reconstruction.RegImageIds();
-  camera_t camera_id = tgt_reconstruction.Image(image_ids[0]).CameraId();
+  const std::set<image_t> image_ids = tgt_reconstruction.RegImageIds();
```

Comment:
Could keep the reference to avoid the copy?

Reply:
No because we cannot modify the set (via DeRegisterImage below) while iterating it.

Reply:
Got it. Didn't see the context below on my phone.